hack/kind/setup.sh

```diff
@@ -78,17 +129,20 @@ setup_cluster() {
 }
 
 install_kubectl() {
-    if ! command -v kubectl &> /dev/null; then
+  header "Installing kubectl"
+
+  if ! command -v kubectl &> /dev/null; then
     info "kubectl not found, attempting to install"
     mkdir -p tmp/bin
     curl -o tmp/bin/kubectl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
```

Comment:
should we consider mac users too? ;) check device type and install based on type of device

Reply:
sure, perhaps in a separate patch.. 
Patches welcome :p

Reply:
I can create a patch for this no worries 😎